docs/source/recipes/Non-streaming-ASR/librispeech/distillation.rst

```diff
@@ -47,7 +47,7 @@ The data preparation contains several stages, you can use the following two
 options:
 
   - ``--stage``
-  - ``--stop-stage``
+  - ``--stop_stage``
```

Comment:
I think --stop-stage and --stop_stage don't make a difference for prepare.sh

Reply:
Ah, I just noticed this, so it should work in both ways. Anyway, this fixes the problem.